omigami/plotting.py

```diff
@@ -7,15 +7,17 @@
 from rdkit.Chem.rdchem import Mol
 import itertools
 import matplotlib.pyplot as plt
+from matplotlib.container import BarContainer
 import requests
 
+from omigami.config import CLASSYFIRE_URL, NPCLASSIFER_URL
+
 
 class MandatoryColumnMissingError(Exception):
     pass
 
 
-# TODO: Dont know about the name though
-class MSPlots:
+class MSPlotter:
```

Comment:
why `MSPlotter` though? why `MS`? 🤔  we are not plotting mass spectra, we are plotting structures and classes and etc no? I'm fine with leaving it like this, I'm just questioning it